sass/mosaico/_form.scss

```diff
@@ -0,0 +1,27 @@
+.crm-mosaico-page {
+  #inputSubject {
+    width: 80%;
+  }
+
+  input[name=campaign] {
+    width: 98%;
+  }
+
+  select {
```

Comment:
please include name values into quotes `&[name="optout_id"]`
same below and above